index.html

```diff
@@ -16,10 +16,11 @@
 </head>
 
 <body>
-    <header class="containerWrap">
+    <header > 
```

Comment:
No space after the header word.
```suggestion
    <header> 
```